android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -170,6 +170,14 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
                 isPersonalizedPrice = call.argument("isPersonalizedPrice"); // TODO: Rename
                 purchasePackage(packageIdentifier, offeringIdentifier, oldSKU, prorationMode, isPersonalizedPrice, result);
                 break;
+            case "purchaseSubscriptionOption":
+                productIdentifier = call.argument("productIdentifier");
+                String optionIdentifier = call.argument("optionIdentifier");
+                oldSKU = call.argument("oldSKU"); // TODO: Rename
+                prorationMode = call.argument("prorationMode"); // TODO: Rename
+                isPersonalizedPrice = call.argument("isPersonalizedPrice"); // TODO: Rename
```

Comment:
This is never sent from the Dart side

Reply:
Not yet 😇 There is a separate ticket for this one but I should probably just do it in this PR

Reply:
are these TODO: Rename still applicable?